src/Codeception/Lib/Interfaces/Web.php

```diff
@@ -47,7 +47,7 @@ public function amOnPage(string $page): void;
      *
      * @param array|string $selector optional
      */
-    public function see(string $text, $selector = null): void;
+    public function see(string $text, array|string $selector = null): void;
```

Comment:
module-webdriver access WebdriverBy as selector too: https://github.com/Codeception/module-webdriver/blob/57e2d8314ee757d1ff8e909afad422402e05f616/src/Codeception/Module/WebDriver.php#L3009

Reply:
Is it really okay to add the `WebDriverBy` union type?
It would seem a bit strange to me that an interface knows so much about its implementations.
In lib-innerbrower it is just `array|string`, so `WebDriverBy` would not make sense as a possible value there.

Is it okay if I make an exception here (and in `dontSee`) and leave it untyped?

Reply:
I left it untyped.